src/View/StringTemplate.php

```diff
@@ -346,7 +346,7 @@ protected function _formatAttribute(string $key, mixed $value, bool $escape = tr
     /**
      * Adds a class and returns a unique list either in array or space separated
      *
-     * @param mixed $input The array or string to add the class to
+     * @param array<string, mixed>|string|null $input The array or string to add the class to
```

Comment:
The actual types include `false` too.

Reply:
Not documented and untested apparently :) I wonder if we should add as "expected inputs".